lib/routes/builds.js

```diff
@@ -292,7 +292,7 @@ module.exports = function (app) {
    * Get build or trigger build for the package.
    *
    * @swagger
-   * /builds/{pkg}:
+   * /builds/{pkg}/{env}:
```

Comment:
`/builds/:pkg` route no longer exists, so it's technically a breaking change. Do we want to keep that route with the optional `env` path that defaults to `dev`?

Reply:
It's already a breaking change due to the status code change from 403 to 409. Ideally, I'd version this as 7.0.0, but @JacopoDaeli is working on the 7.x branch for the new codebase. We can choose to just rev that to 8.x and ship this as a major version, or we can choose to just modify this as-yet undocumented API and ship as a minor. I'm fine either way.

Reply:
Let's ship this a minor please :)